### DISCUSSION TITLE
Help!. How can I get the data that is obtained from "Human"?

### DISCUSSION THREAD

**Discussion by pooldeveloper** [vladmandic/human#215] [Q&A] [answered]
I need to get the displayed data from the canvas to another clean app.js file
![image](https://user-images.githubusercontent.com/74687129/141647216-aa64b6cc-b266-417b-8982-70e5ac701125.png)

I'm going to "human.esm.js" and I'm extracting it with a function that is exported from my new "app.js" file, but I don't think it's the best way to do it.
![image](https://user-images.githubusercontent.com/74687129/141647311-5dbf8e0e-628c-4f3c-87c6-26a88e1b72e1.png)

I hope you can help me since the documentation is not here. Thanks.

**Comment by vladmandic** [comment, marked as answer]
not sure whats the problem, simply read it from `human.result.gesture`
and documentation for gestures is <https://github.com/vladmandic/human/wiki/Gesture> and <https://vladmandic.github.io/human/typedoc/interfaces/Result.html#gesture>

**Reply by pooldeveloper** [reply]
Great it helped me to read that :)
